labs/coding-101-rest-basics-ga/2.md

```diff
@@ -40,6 +40,6 @@ In this example, you request the list of hosts, and that information is returned
 
 (<a href="http://www.json.org/" target="_blank">JSON</a> -- JavaScript Object Notation, is a lightweight text-based open standard designed for human-readable data interchange.)
 
-(<a href="http://https://www.w3schools.com/xml/xml_whatis.asp/" target="_blank">XML</a> -- Extensible Markup Language, a language designed to store and transport data.)
+(<a href="https://www.w3schools.com/xml/xml_whatis.asp/" target="_blank">XML</a> -- eXtensible Markup Language, a language designed to store and transport data.)
```

Comment:
Sorry, this link gives a 404 with that trailing `/` - change `https://www.w3schools.com/xml/xml_whatis.asp/` to `https://www.w3schools.com/xml/xml_whatis.asp` and it works.